docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -198,7 +251,13 @@ EOF
 terraform apply
 ----
 
-. Create the shown DNS records
+. Create the shown DNS records and wait for them to propagate!
++
+[source,console]
+----
+sleep 600
+host "api.${CLUSTER_ID}.${BASE_DOMAIN}"
```

Comment:
```suggestion
while -n 15 host "api.${CLUSTER_ID}.${BASE_DOMAIN}"
```

Not sure if that really helps. I have the habit to run such test commands using while.

Reply:
Unfortunately the command doesn't work like this?